modin/experimental/core/execution/native/implementations/omnisci_on_native/io/io.py

```diff
@@ -466,7 +466,9 @@ def _read_csv_check_support(
             )
 
         parse_dates_unsupported = isinstance(parse_dates, dict) or (
-            isinstance(parse_dates, list) and isinstance(parse_dates[0], list)
+            isinstance(parse_dates, list)
+            and len(parse_dates) > 0
+            and all(isinstance(date, list) for date in parse_dates)
```

Comment:
```suggestion
            and any(isinstance(date, list) for date in parse_dates)
```
I think this is maybe supposed to be `any`? Would `parse_dates=[['col1', 'col2'], 'col3']` be supported?